rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -47,7 +49,7 @@ pub enum ErrorKind {
     #[error(transparent)]
     InvalidRequestArgument(#[from] rpc_command::invoker::RequestBuilderError),
     /// Client Id used for the ADR Client was invalid.
-    #[error("{0}")]
+    #[error("{0} is invalid as the identifier for the client")]
```

Comment:
```suggestion
    #[error("{0} is invalid as client id")]
```
just to keep consistent terminology